execution-scheduler/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup
+
+setup(
+    name='cloudify-execution-scheduler',
+    version='5.2.0.dev1',
+    author='Cloudify',
+    author_email='cosmo-admin@cloudify.co',
+    packages=['execution_scheduler'],
+    entry_points={
+        'console_scripts': [
+            'cloudify-execution-scheduler = execution_scheduler.main:main',
```

Comment:
this will just run the `main` function, ie. NOT the `if __name__` block, and that block has some important setup